build fail real:
```
======================================================================
ERROR: test_archived_form_gets_removed_from_case_xform_ids (corehq.apps.commtrack.tests.test_xml.BugSubmissionsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/commtrack/tests/test_xml.py", line 464, in test_archived_form_gets_removed_from_case_xform_ids
    timestamp=datetime.utcnow() + timedelta(-30)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/test/utils.py", line 216, in inner
    return test_func(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/commtrack/tests/test_xml.py", line 194, in submit_xml_form
    **submit_extras
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/receiverwrapper/util.py", line 24, in submit_form_locally
    **kwargs
  File "corehq/ex-submodules/couchforms/util.py", line 330, in run
    instance, cases, errors = self.process_xforms_for_cases(lock_manager)
  File "corehq/ex-submodules/couchforms/util.py", line 356, in process_xforms_for_cases
    stock_result = process_stock(xforms, case_db)
  File "submodules/dimagi-utils-src/dimagi/utils/decorators/log_exception.py", line 14, in decorated
    return fn(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/commtrack/processing.py", line 221, in process_stock
    actions_for_form = get_stock_actions(xform)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/commtrack/processing.py", line 180, in get_stock_actions
    case_action_intents = _get_case_action_intents(transaction_helpers)
TypeError: _get_case_action_intents() takes exactly 2 arguments (1 given)
```

--- a/corehq/apps/commtrack/processing.py
+++ b/corehq/apps/commtrack/processing.py
@@ -177,8 +177,9 @@ def get_stock_actions(xform):
     if not transaction_helpers:
         return _empty_actions()
 
-    case_action_intents = _get_case_action_intents(transaction_helpers)
+    case_action_intents = _get_case_action_intents(xform, transaction_helpers)
     return StockFormActions(stock_report_helpers, case_action_intents)
+
 
 def _get_case_action_intents(xform, transaction_helpers):
     # list of cases that had stock reports in the form

--- a/corehq/apps/commtrack/processing.py
+++ b/corehq/apps/commtrack/processing.py
@@ -264,7 +264,7 @@ def mark_cases_changed(case_action_intents, case_db):
                 case.xform_ids.append(action_intent.form_id)
 
         case_db.mark_changed(case)
-
+    return relevant_cases
 
 def _compute_ledger_values(original_balance, stock_transaction):
     if stock_transaction.report.type == stockconst.REPORT_TYPE_BALANCE:

